pkg/lib/authenticationflow/declarative/data_identification.go

```diff
@@ -32,6 +32,8 @@ type IdentificationOption struct {
 	Alias string `json:"alias,omitempty"`
 	// WechatAppType is specific to OAuth.
 	WechatAppType wechat.AppType `json:"wechat_app_type,omitempty"`
+	// ProviderStatus is specific to OAuth.
+	ProviderStatus OAuthProviderStatus `json:"provider_status,omitempty"`
```

Comment:
If we call this `provider_status`, then we should also call it `provider_status` in the previous commit. So that the names are consistent.